libraries/AP_Avoidance/AP_Avoidance.h

```diff
@@ -156,7 +156,7 @@ class AP_Avoidance {
     void deinit();
 
     // get unique id for adsb
-    uint32_t src_id_for_adsb_vehicle(AP_ADSB::adsb_vehicle_t vehicle) const;
+    uint32_t src_id_for_adsb_vehicle(const struct AP_ADSB::adsb_vehicle_t &vehicle) const;
```

Comment:
it's slightly funny to have the "struct" in the declaration but not in the implementation.  I'd remove it from the declaration.